.ci/setup_headless_display.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+set -x
+
+# sudo apt-get update && sudo apt-get install python-qt4 libgl1-mesa-glx
+sudo apt-get install python-qt4 libgl1-mesa-glx
+sudo apt-get install -y xvfb
+export DISPLAY=:99.0
+export PYVISTA_OFF_SCREEN=True
+which Xvfb
+Xvfb :99 -screen 0 1024x768x24 > /dev/null 2>&1 &
+# Debugging commands:
+# ls -l /etc/init.d/
+# sh -e /etc/init.d/xvfb start
+# give xvfb some time to start
+sleep 3
+set +x
```

Comment:
We should clone https://github.com/pyvista/gl-ci-helpers instead and maybe change its directories from `tavis` and `appveyor` to `linux` and `windows`

Reply:
Changing those directories will have consequences for a few other projects so we should add aliases to those old files as well.

Reply:
This is why I'm fine with having it here as there's no dependency requirements.  I vote it stays here.

Reply:
Fair enough. Sounds good to me then